scene/3d/visual_instance_3d.cpp

```diff
@@ -120,6 +120,12 @@ bool VisualInstance3D::is_sorting_use_aabb_center() const {
 	return sorting_use_aabb_center;
 }
 
+void VisualInstance3D::_validate_property(PropertyInfo &p_property) const {
+	if ((p_property.name == "sorting_offset" || p_property.name == "sorting_use_aabb_center")) {
```

Comment:
You have an extra pair of parentheses here and below.

This also seems to require a classref sync, the default value for `sorting_use_aabb_center` in VisualInstance3D is no longer picked up by doctool. (We could maybe fix that but it's not critical for this PR.)